server/middleware/authorisationMiddleware.test.ts

```diff
@@ -8,7 +8,7 @@ function createToken(authorities: string[]) {
     user_name: 'USER1',
     scope: ['read', 'write'],
     auth_source: 'nomis',
-    ...authorities,
+    authorities,
```

Comment:
this is a behaviour change i think ? but the previous code looks broken ?

Reply:
When added types to the params `createResWithToken`, I also changed the structure of the arguments, so the `authorities` param went from being the passed-in object to being the `authorities` property value of the passed-in object. So the corresponding change was needed here.